include/pure_pursuit.h

```diff
@@ -57,6 +58,218 @@ class PurePursuit{
 	*/
 	std::pair<Point2D,double> get_location_on_path(const Point2D& state){
 	//Seach for the shortest distance from the robot to each path segment
+		double counter = 0;
+		double slope, distanceTraveled, distanceToRobot, shortestDistance = INT16_MAX;
```

Comment:
Move functions implementations to separate cpp files. Only definitions should be in a header file.